hack/e2e.sh

```diff
@@ -2,22 +2,24 @@
 
 set -euo pipefail
 
-# HACK: Update the devstack default provider network name to match the one
-# hardcoded in the cirros example
-export OS_CLOUD=devstack-admin
-openstack network set --name provider_net_dualstack_1 private
+kubectl kuttl test
 
-# Now drop admin privileges
-export OS_CLOUD=devstack
+# # HACK: Update the devstack default provider network name to match the one
+# # hardcoded in the cirros example
+# export OS_CLOUD=devstack-admin
+# openstack network set --name provider_net_dualstack_1 private
 
-cd examples
+# # Now drop admin privileges
+# export OS_CLOUD=devstack
 
-# Populate example credentials
-sed "s/  devstack:/  openstack:/g" /etc/openstack/clouds.yaml > credentials/clouds.yaml
-make load-credentials
+# cd examples
 
-# Apply the cirros server example and wait for the server to be available
-kubectl apply -k apply/cirros --server-side
-kubectl wait --timeout=10m --for=condition=available server ${USER}-cirros-server
+# # Populate example credentials
+# sed "s/  devstack:/  openstack:/g" /etc/openstack/clouds.yaml > credentials/clouds.yaml
+# make load-credentials
 
-openstack server show "$(kubectl get server ${USER}-cirros-server -o jsonpath='{.status.id}')"
```

Comment:
Is there any reason we can't continue to run this example, at least for now?

Reply:
This isn't supposed to go in the final PR. We'll continue to exercise the cirros example as we did before. Actually, we may want to exercise _all_ of the examples in the future.